modules/social_features/social_core/config/update/social_core_update_10200.yml

```diff
@@ -0,0 +1,30 @@
+field.field.block_content.basic.field_text_block:
```

Comment:
Since we add this to 10.3 milestone, need to rename the number to start with 103

Reply:
fixed